### ISSUE TITLE
Feature: Fatalities per 10'000

### ISSUE THREAD

**Issue by TC827** [daenuprobst/covid19-cases-switzerland#70]
"Fatalities per 10'000" is the single most important value in my opinion.
The criterion to test or not to test a person depend on the Canton, but a dead person is a dead person every where (I like to think, that at least in Switzerland statistics are not adjusted in order let the other believe that we are the best of the world).

To be put as radio button in the first Switzerland map (with the Cantonal data).

**Comment by chatelao** [comment]
I totally agree - the top-value of Zurich in the heatmap is not very helpful compared to a cases/100k heatmap showing the normalized "hot spots".

Else: Great Website, love it.
